smarthint/loaders/PLPBanners.ts

```diff
@@ -0,0 +1,114 @@
+import { AppContext } from "../mod.ts";
+import { Banner, FilterProp, SearchSort } from "../utils/typings.ts";
+import { getSessionCookie } from "../utils/getSession.ts";
+import { getFilterParam, getSortParam } from "../utils/transform.ts";
+import { ProductListingPage } from "../../commerce/types.ts";
+import { getCategoriesParam } from "./recommendations.ts";
+
+export type RuleType = "valuedouble" | "valuedate" | "valuestring";
```

Comment:
interesting, is the word `value` needed?

Reply:
Yes. https://developers.smarthint.co/#ea67ec69-4c0c-4004-8b6d-50f1c016d2e9